src/main/java/org/broadinstitute/hellbender/utils/test/BaseTest.java

```diff
@@ -62,10 +62,10 @@ public void setTestVerbosity(){
     public static final String publicTestDir = new File(gatkDirectory, publicTestDirRelative).getAbsolutePath() + "/";
     public static final String publicTestDirRoot = publicTestDir.replace(publicTestDirRelative, "");
 
-    public static final String GSC_GATK_TEST_RESOURCES = "gs://hellbender/test/resources/";
+    public static final String GCS_GATK_TEST_RESOURCES = "gs://hellbender/test/resources/";
```

Comment:
`Basetest.getGCPTestInputPath()` returns the appropriate test path.  (Maybe it should be changed to GCS?  I can't remember what gcp stood for?